exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,14 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if (grade < 50) {
+    return "failed"
+  }
+  else if (grade >= 50) {
+    return "passed"
+  }
+}
```

Comment:
Likewise, you can simplify your function into: 
function studentPassed(grade) {
  if (grade < 50){
    return "failed";
  }else{
    return "passed";
  }
}